admin_solutions/certificate_management.adoc

```diff
@@ -0,0 +1,73 @@
+[[admin-solutions-certificate-management]]
+= Certificate Management
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+Over the lifetime of a {product-title} cluster, certificates will enter various
+phases of their lifecycle.  The following procedures describe how to manage
```

Comment:
The tiniest nit: there's a double space before, "The following procedures describe..."

Otherwise, LGTM! 🌠